titus-client/src/main/java/com/netflix/titus/runtime/endpoint/v3/grpc/GrpcJobManagementModelConverters.java

```diff
@@ -400,13 +402,13 @@ private static ContainerHealth toCoreContainerHealth(com.netflix.titus.grpc.prot
     }
 
     private static List<ContainerState> toCoreContainerState(List<com.netflix.titus.grpc.protogen.TaskStatus.ContainerState> containerStates) {
-        List<ContainerState> containerStateList = null;
+        List<ContainerState> containerStateList = Collections.emptyList();
```

Comment:
Does this work? I thought `emptyList()` was immutable, which implies that the lines below won't be able to `add()`

Reply:
I think you just want `new ArrayList<>();`

Reply:
probably, I am testing it now